project/Dependencies.scala

```diff
@@ -14,12 +14,11 @@ object Dependencies {
   val AlpakkaKafkaVersionInDocs = "2.0"
 
   object Versions {
-    val akka = sys.props.getOrElse("build.akka.version", "2.6.6")
-    val alpakka = "2.0.0"
+    val akka = sys.props.getOrElse("build.akka.version", "2.6.7")
```

Comment:
We can probably hold on the Akka version.

Reply:
2.6.7 has some improvements (`askWithStatus`) that we can use in our documentation. We also rely on some ApiMayChange parts of Akka (`ShardedDaemonProcess`) so it's good to keep follow the Akka releases for now.